pyproject.toml

```diff
@@ -1,9 +1,47 @@
 [build-system]
-requires = [
-    "setuptools",
-    "wheel",
+requires = ["setuptools"]
+
+[project]
+name = "asteroid-filterbanks"
+authors = [
+    {name = "Manuel Pariente", email="manuel.pariente@loria.fr"}
```

Comment:
Change to pariente.mnl@gmail.com please.

Reply:
Done